src/frontend/components/Felleskomponenter/BlokkerTilbakeKnappModal/BlokkerTilbakeKnappModal.tsx

```diff
@@ -22,45 +22,38 @@ const StyledEksternLenke = styled(EksternLenke)`
 `;
 
 const BlokkerTilbakeKnappModal = () => {
-    const [show, setShow] = useState(false);
+    const { visBlokkerTilbakeKnappModal, settVisBlokkerTilbakeKnappModal } = useAppNavigation();
     const { formatMessage } = useIntl();
 
-    const håndterNavigasjon = () => {
-        setShow(true);
-        return false;
-    };
     const håndterAvbryt = () => {
-        setShow(false);
+        settVisBlokkerTilbakeKnappModal(false);
     };
 
     return (
-        <>
-            <Prompt message={håndterNavigasjon} />
-            <Modal
-                onClose={() => setShow(false)}
-                open={show}
-                aria-label={formatMessage({ id: 'felles.blokkerTilbakeKnapp.modal.tittel' })}
-            >
-                <ModalContent>
-                    <Heading level={'1'} size={'large'}>
-                        <SpråkTekst id={'felles.blokkerTilbakeKnapp.modal.tittel'} />
-                    </Heading>
-                    <BodyLong>
-                        <SpråkTekst id={'felles.blokkerTilbakeKnapp.modal.tekst'} />
-                    </BodyLong>
-                    <Flex>
-                        <StyledEksternLenke
-                            lenkeSpråkId={'kvittering.dinesaker.lenke'}
-                            lenkeTekstSpråkId={'felles.blokkerTilbakeKnapp.modal.tilDittNavKnapp'}
-                            target="_blank"
-                        />
-                        <Button onClick={håndterAvbryt}>
-                            <SpråkTekst id={'felles.blokkerTilbakeKnapp.modal.avbrytKnapp'} />
-                        </Button>
-                    </Flex>
-                </ModalContent>
-            </Modal>
-        </>
+        <Modal
```

Comment:
Hovedsaklig fjernet Prompt. Se PR for ks-soknad: https://github.com/navikt/familie-ks-soknad/pull/219